.gitlab/new-e2e_testing/windows.yml

```diff
@@ -48,3 +62,14 @@ new-e2e-windows-agent-msi-windows-server-a7-x86_64:
   rules:
     - !reference [.on_default_new-e2e_tests_a7]
     - !reference [.on_windows_installer_changes_or_manual]
+
+new-e2e-windows-agent-domain-tests-a7-x86_64:
+  stage: kitchen_testing
+  extends:
+    - .new_e2e_template
+    - .new-e2e_windows_domain_test
+    - .new-e2e_windows_a7_x86_64
+    - .new-e2e_agent_a7
+  rules:
+    - !reference [.on_default_new-e2e_tests_a7]
```

Comment:
`on_default` will run this job on every PR, is that what you meant? If you just want the job to run on main/mergequeue use `.on_deploy_a7`

Reply:
I'm fine either way just wanted to make sure it was intentional and not copy+paste since it's different than what the kitchen tests did I think.
Also `.on_windows_installer_changes_or_manual` should ensure the tests run if any relevant files are changed.